README.md

```diff
@@ -1,2 +1,99 @@
-# element-template-validator
-Element Template Validation Tool based on JSON Schema files
+# element-templates-validator
```

Comment:
Is this project simply a tiny wrapper around a JSON schema validator? Or does it contain any element template specific things? Otherwise we might simply call it "json-schema-validator".

Reply:
Makes sense 👍